src/VirtoCommerce.CustomerExportImportModule.Data/VirtoCommerce.CustomerExportImportModule.Data.csproj

```diff
@@ -19,15 +19,16 @@
         <Folder Include="Handlers\" />
         <Folder Include="Models\" />
         <Folder Include="Repositories\" />
-        <Folder Include="Services\" />
     </ItemGroup>
 
     <ItemGroup>
+        <PackageReference Include="CsvHelper" Version="12.2.2" />
         <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.8">
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+        <PackageReference Include="VirtoCommerce.CustomerModule.Core" Version="3.28.0" />
```

Comment:
Seems like redundant dependency

Reply:
I use Customer module services and models here...

Reply:
Fixed